File: examples/php/basic_usage.php
#!/usr/bin/env php
/**
 * Basic usage example for HiTechCloud Php SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_API_KEY=your-api-key
 */
<?php
use HiTechCloud\HiTechCloud;

$client = new HiTechCloud('your-api-key');

// List all services
$services = client->services()->listServices();
foreach ($services['data'] as $svc) {
    echo $svc['id'] . ': ' . $svc['name'] . PHP_EOL;
}

// Get a specific service
$service = client->services()->getService(['id' => 123]);
print_r($service);

// Power operations
client->services()->startService(['id' => 123]);
client->services()->stopService(['id' => 123]);
client->services()->rebootService(['id' => 123]);

File: examples/go/basic_usage.go
#!/usr/bin/env go
/**
 * Basic usage example for HiTechCloud Go SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_API_KEY=your-api-key
 */
package main

import (
    "github.com/hitechcloud-vietnam/hitechcloud-sdk/sdk/go"
)

func main() {
    client := hitechcloud.NewClient("your-api-key")
}

// List all services
services, err := client.Services.ListServices(nil)
if err != nil {
    log.Fatal(err)
}
fmt.Println(services)

// Get a specific service
service, err := client.Services.GetService(123)
if err != nil {
    log.Fatal(err)
}
fmt.Println(service)

// Power operations
client.Services.StartService(123)
client.Services.StopService(123)
client.Services.RebootService(123)
